core/src/Authorization.html

```diff
@@ -10,10 +10,15 @@
 
 <script type="text/javascript">
   import { LuigiConfig } from './services/config.js';
-  import { isFunction, isPromise, deepMerge, getConfigValueFromObject } from './utilities/helpers.js';
+  import {
+    isFunction,
+    isPromise,
+    deepMerge,
+    getConfigValueFromObject
+  } from './utilities/helpers/helpers.js';
```

Comment:
I would love to see `helpers.js` being renamed to something more meaningful. `basic-helpers` or maybe something better?

Reply:
`generic-helpers`?

Reply:
ok